README.md

```diff
@@ -300,7 +300,7 @@ fastify.listen(3000, (err) => {
 async function validateToken(request, decodedToken) {
   const blacklist = ['token1', 'token2']
 
-  return blacklist.includes(decodedToken.jti)
+  return !blacklist.includes(decodedToken.jti)
```

Comment:
can you also rename from blacklist to denylist while you are at it.

Reply:
absolutely 👍🏽